libs/api/src/lib/graphql/event/event.query.ts

```diff
@@ -129,12 +131,48 @@ const createToFilter = (filter?: Partial<EventFilter>): Prisma.EventWhereInput =
   return {}
 }
 
+const createNameFilter = (filter?: Partial<EventFilter>): Prisma.EventWhereInput => {
+  if (filter?.name) {
+    return {
+      OR: [
+        {
+          name: {
+            contains: filter.name,
```

Comment:
Could full text search in these maybe work better than contains?

Reply:
if you mean `search` instead of `contains`, then it didn't seem to work better - it seemed like it searched by the beginning of the word